django/contrib/admin/options.py

```diff
@@ -339,11 +339,11 @@ def lookup_allowed(self, lookup, value):
         # Check FKey lookups that are allowed, so that popups produced by
         # ForeignKeyRawIdWidget, on the basis of ForeignKey.limit_choices_to,
         # are allowed to work.
-        for l in model._meta.related_fkey_lookups:
+        for L in model._meta.related_fkey_lookups:
```

Comment:
Hi @timgraham, i think you use https://pypi.python.org/pypi/flake8 latest update `2016-08-08`,
Yes, there is a warnings `E305 expected 2 blank lines after class or function definition, found 1` in many files,you can allow to me for fix it?
https://code.djangoproject.com/ticket/27463#comment:10